use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java

```diff
@@ -45,22 +42,21 @@ public class VehicleRouteDemoResource {
     private static final LocalTime AFTERNOON_WINDOW_END = LocalTime.of(18, 0);
 
     public enum DemoData {
-        PHILADELPHIA(0, 60, 6, 2, LocalTime.of(7, 30),
+        PHILADELPHIA(0, 55, 6, LocalTime.of(7, 30),
```

Comment:
Was the change to 55 made intentionally?

Reply:
Yes, given that there are now more homeLocations than there were depots, the solver failed to find a feasible solution with 60 visits. Reducing it to 55 visits makes sure that there is a feasible solution + gives some bandwidth for planning recommended fit visits